src/Uno.UI/UI/Xaml/ResourceResolver.cs

```diff
@@ -61,14 +61,27 @@ static ResourceResolver()
 		/// Performs a one-time, typed resolution of a named resource, using Application.Resources.
 		/// </summary>
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public static T ResolveResourceStatic<T>(object key, object context = null)
+		public static object ResolveResourceStatic(object key, Type type, object context = null)
```

Comment:
Keep the generic method as well, for backwards binary compatibility. (It may be used by libraries compiled against older versions of Uno)